packages/flutter_availability/lib/src/config/availability_options.dart

```diff
@@ -45,8 +46,13 @@ class AvailabilityOptions {
   /// A way to provide your own secondary button implementation
   final ButtonBuilder secondaryButtonBuilder;
 
-  /// A way to provide your own text button implementation
-  final ButtonBuilder textButtonBuilder;
+  /// A way to provide your own big text button implementation
+  /// This is used as a tertiary button
+  final ButtonBuilder bigTextButtonBuilder;
+
+  /// A way to provide your own small text button implementation
+  /// This is used as a tertiary button smaller variant
```

Comment:
Suggestion of better text: "This is used as a smaller variant of the tertiary button"